fluent-tests/src/test/java/com/azure/mgmttest/RuntimeTests.java

```diff
@@ -480,4 +482,30 @@ private static <T> void assertMethodExist(Class<T> clazz, String methodName, Str
             Assertions.fail("Method should exist: " + clazz.getName() + " " + methodName + "(" + parametersSignature + ")");
         }
     }
+
+    @Test
+    public void testStreamStyleSerialization() throws IOException {
+        SerializerAdapter serializerAdapter = SerializerFactory.createDefaultManagementSerializerAdapter();
+        Map<String, Object> pirCommunityGalleryResourceMap = new HashMap<>() {{
+            this.put("name", "myName");
+            this.put("location", "myLocation");
+            this.put("type", "myType");
+        }};
```

Comment:
nit, `Map.of`

Reply:
Personally I'd prefer create the JSON via a JSON lib .e.g `JsonNode` (instead of serialize a `Map`)

Reply:
Makes sense. Changed to use json string.